tests/dec/test_dec.py

```diff
@@ -191,6 +193,25 @@ def test_missing_global_photos_flag():
     assert not p.global_photos_flag()
 
 
+def test_model_aliases():
+    p = DecFileParser(DIR / "../data/defs-aliases-chargeconj.dec")
+    p.parse()
+
+    assert len(p.dict_model_aliases()) == 7
+    assert p.dict_model_aliases()["SLBKPOLE_DtoKlnu"] == [
```

Comment:
I'm confused here. I understood that `dict_model_aliases()` is providing a dict, not a Lark object "unmassaged". I was then expecting something similar to what you see for example in
```
def test_pythia_definitions_parsing():
    p = DecFileParser(DIR / "../data/defs-aliases-chargeconj.dec")
    p.parse()

    assert p.dict_pythia_definitions() == {
        "ParticleDecays:mixB": "off",
        "Init:showChangedSettings": "off",
        "Init:showChangedParticleData": "off",
        "Next:numberShowEvent": 0.0,
    }


def test_jetset_definitions_parsing():
    p = DecFileParser(DIR / "../data/defs-aliases-chargeconj.dec")
    p.parse()

    assert p.dict_jetset_definitions() == {
        "MSTU": {1: 0, 2: 0},
        "PARU": {11: 0.001},
        "MSTJ": {26: 0},
        "PARJ": {21: 0.36},
    }
```

Reply:
I totally understand the good reason of avoiding a packing-unpacking sequence. But then one would separate what are internal manipulations and what are functions that provide info to the user in a friendly way. Maybe here you would need mostly internal functions and the public one would give a dict of alias:model names only. Let me know what you think.

Otherwise it seems we're good to go.

Reply:
That seems like a good idea, I'll split this into public and private methods.

Reply:
Okay, I've now done that. For some reason the ` _parsed_dec_file` in `DecFileParser` was annotated with `str` instead of `Tree` which mypy didn't accept. I've changed it to `Tree | None`, please let me know if there was a reason for the previous behavior and I'll revert this.

And thank you for finding time to review this!

Reply:
Nope, strange that mypy did not complain before. I won't investigate, though.